cmd/podman/container.go

```diff
@@ -15,7 +15,12 @@ var containerCommand = cliconfig.PodmanCommand{
 	},
 }
 
+var containerCommands = []*cobra.Command{
```

Comment:
Why is this separate from the rest? The single exists command, and then `getContainerSubCommands()` separately?

Reply:
added a comment but for the record, this is the way the old cli was too. commands that both the remote client and local client have implemented are there.